### PR TITLE
Added changes of previous PR

### PR DESCRIPTION
**Pull request by alexawesomecode**
I created a new PR with stickler checking in this one.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,625 @@
...
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css" />
+    <title>Newsweek</title>
+
+</head>
+<main>
```
Comment (Redvanisation) [inline index.html]: The `<main>` tag should be inside the `<body>` tag and it should wrap the main content of the website only, for the headings, and the footer they should be wrapped in their appropriate tags too: `<header>` and `<footer>`.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @alexawesomecode  🌟

**Good work on this project, however, you still have to fix the following ⬇️** 

1. The clone is not responsive, it should look like the original website on different screen sizes, check the screenshot below: 
Screen Shot 2019-10-17 at 6 19 40 PM

2. Please fix the semantic HTML tags as mentioned in my comment below.

3. The footer should have social media icons and main navigation links wrapped in anchor tags `<a>`.

4. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input).

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by ebukaume** [comment]
Hi @alexawesomecode :wave: 

Good job so far, however, there are some issues to address.

**Navigation bar**
- [ ] It contains Newsweek logo. (It should be a logo not heading `consider using `)
- [ ] It contains Sign in and Subscribe links. (`They should be links and both different elements`)
- [ ] It contains main navigation links. (These should be links `I believe <a> is the right tag to use here`)

**Footer**
* [ ] It contains Newsweek logo. (Same as the one in the navbar)
* [ ] It contains social media icons. (Consider making these links since they are meant to refer users to another resource `social media page`)

**Kindly submit another code review request after fixing the above issues.**


Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Comment by ebukaume** [comment]
## Congratulations! :champagne: 
Hi @alexawesomecode  :wave: 

Congratulations. You can go ahead and merge this branch into master. I wish you good luck with your next projects.

Happy Coding :v: 
**Ebuka Umeokonkwo**
 [Github](https://github.com/ebukaume) | [Linkedin](https://www.linkedin.com/in/ebukaume) | [Slack](https://app.slack.com/team/UFDKXE68L) | [Twitter](https://twitter.com/ebukaume)

**Pull request merged**